src/scripts/libs/snippets.js

```diff
@@ -0,0 +1,3 @@
+// from Material Design Lite website
```

Comment:
Would you be open to using this instead: https://github.com/gauntface/gf-site/blob/master/src/frontend/scripts/helpers/clipboard.js

May need some tweaking to use module.exports, but would allow the use of eslinting again.